apps/openmw/mwrender/sky.cpp

```diff
@@ -322,11 +321,12 @@ class ModVertexAlphaVisitor : public osg::NodeVisitor
     int mMeshType;
 };
 
-class CelestialBody
+// A base class for the sun and moons. Must be stored in an osg::ref_ptr due to being
+// derived from osg::Referenced.
```

Comment:
Have you considered changing these to Doxygen style comments?
Like this (note the extra '/'):
/// A base class for the sun and moons.
/// \note Must be stored in an osg::ref_ptr due to being derived from osg::Referenced.